Dockerfile

```diff
@@ -11,6 +10,7 @@ RUN apk add -U --no-cache git binutils
 RUN go get github.com/nats-io/nats-top
 
 RUN go get -u -ldflags "-X main.version=0.3.16-nats-box" github.com/nats-io/nsc
+RUN go get -u -ldflags "-X main.version=0.3.16-nats-box" github.com/nats-io/jetstream/nats
```

Comment:
what decides this version string? Seems a bad way to do this is all has to be the version of nats box and not that of he upstream

Reply:
Yes would ideally be the version of upstream, should be revisited...